spec/helpers/passengers_helper_spec.rb

```diff
@@ -37,9 +37,15 @@
         expect(helper.passengers_table_row_class(@passenger)).to eql 'inactive'
       end
     end
-    context 'expiration date blank' do
+    context 'expiration overridden' do
+      it "returns 'overridden'" do
+        @doctors_note.update override_expiration: true
+        expect(helper.passengers_table_row_class(@passenger)).to eql 'overridden'
+      end
+    end
+    context 'no note' do
       it "returns 'no_note'" do
-        @doctors_note.destroy
+        @passenger = create :passenger, :temporary, :no_note
```

Comment:
Can you explain this? I could make an educated guess why you made this change, but I just want to be sure.

Reply:
I thought destroying the object (my first solution) seemed a little roundabout. Clearer to state "okay now we want the passenger to have no note."

Reply:
Great!